roles/subscription-manager/tasks/offline.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: get repo file
+  get_url:
+    url: "http://{{hostvars['wiley-kragle']['ansible_default_ipv4']['address']}}/offline.repo"
```

Comment:
We can do that, I was trying to make it as dynamic as possible, but I guess having the name of the node is the same kind of thing.  I'll make that change and push
good catch.